### DISCUSSION TITLE
和 Projects 插件结合的场景

### DISCUSSION THREAD

**Discussion by milla** [zhaohongxuan/obsidian-weread-plugin#130] [Show and tell]
场景1: Gallery 
<img width="1429" alt="image" src="https://user-images.githubusercontent.com/297531/215077532-3b72f5cb-8dd4-41d4-8fea-c8231a767d3a.png">

场景2: 阅读日历
<img width="1325" alt="image" src="https://user-images.githubusercontent.com/297531/215078123-2064a45f-3e2f-4e1e-8ae4-09b5b0de7155.png">

场景3: 分类板块（目前没有status prop，所以场景不明确）
<img width="1336" alt="image" src="https://user-images.githubusercontent.com/297531/215077909-5f403476-85c5-4d04-8f77-6b506f407870.png">

Projects 插件
https://github.com/marcusolsson/obsidian-projects

**Comment by robinguo** [comment]
阅读日历怎么做出来的？
